## 하는 김에 성능테스트..후훗


In [2]:
import numpy as np
L  = range(1000)

In [5]:
%timeit [i**2 for i in L]

134 µs ± 168 ns per loop (mean ± std. dev. of 7 runs, 10000 loops each)


#### 회사 gpu서버 vm 성능
#### 196 us +- 253 ns per loop  (mean ± std. dev. of 7 runs, 10000 loops each)

####  집 서버 컴  xeon3서버 E3-1230v2 32기가램.
#### 263 µs ± 49.7 ns per loop (mean ± std. dev. of 7 runs, 1000 loops each)

#### 집 GPU 서버:: AMD 5950x 128G
#### 134 µs ± 168 ns per loop (mean ± std. dev. of 7 runs, 10000 loops each)




In [7]:
N = np.arange(1000)
%timeit N**2

693 ns ± 1 ns per loop (mean ± std. dev. of 7 runs, 1000000 loops each)
